mmdet/apis/train.py

```diff
@@ -180,6 +180,8 @@ def train_detector(model,
 
     # fp16 setting
     fp16_cfg = cfg.get('fp16', None)
+    if fp16_cfg is None and cfg.get('device', None) == 'npu':
+        fp16_cfg = dict(loss_scale=512.)
```

Comment:
dynamic is better ?

Reply:
In MMCV, when using AMP in Torch, the loss_scale setting dynamic of FP16 only defaults to the initial value to 65536, and using a smaller as default initial value can avoid the incorrect update of the layer containing the buffer layer caused by excessive overflow at the beginning of training